client/package.json

```diff
@@ -26,12 +26,15 @@
     "create-react-app": "^5.0.0",
     "css-loader": "^6.7.1",
     "env-cmd": "^10.1.0",
+    "graphiql": "^2.4.0",
     "graphql": "^15.5.1",
+    "graphql-playground-react": "^1.7.28",
```

Comment:
if we're serving it all in the frontend now, we need to pull all of the stuff out of the backend

Reply:
this change has been made